virttest/qemu_storage.py

```diff
@@ -915,6 +916,7 @@ def check_image(self, params, root_dir, force_share=False):
                                               " during image check. No data "
                                               "integrity problem was found "
                                               "though. (%s)" % image_filename)
+                return cmd_result
```

Comment:
Please add the docstring `:return:` for ` return cmd_result`, since this is module-level interface.
And I thought you might add `return` in the `else` situation for consistency.

Thanks.

Reply:
Updated, thanks.